tests/api_connexion/conftest.py

```diff
@@ -41,7 +42,31 @@ def minimal_app_for_api():
     )
     def factory():
         with conf_vars({("api", "auth_backends"): "tests.test_utils.remote_user_api_auth_backend"}):
-            return app.create_app(testing=True, config={"WTF_CSRF_ENABLED": False})  # type:ignore
+            _app = app.create_app(testing=True, config={"WTF_CSRF_ENABLED": False})  # type:ignore
+            init_mock_cors_middleware(_app, allow_origins=["http://apache.org", "http://example.com"])
```

Comment:
Just trying to understand. Why these 2 URLs are allowed?

Reply:
![Screenshot from 2024-03-25 20-15-32](https://github.com/sudiptob2/airflow/assets/73622805/9065e17d-771a-4f4e-ad9b-c2ca08cf0ede)

Here is the original testing code. They registered the two domains here.